pypass/passwordstore.py

```diff
@@ -61,22 +61,44 @@ def __init__(
             path=os.path.join(os.getenv("HOME"), ".password-store"),
             git_dir=None,
     ):
-        self.path = path
+        self.path = os.path.abspath(path)
 
-        # Read the .gpg-id
-        gpg_id_path = os.path.join(path, '.gpg-id')
-        if os.path.isfile(gpg_id_path):
-            with open(gpg_id_path, 'r') as gpg_id_file:
-                self.gpg_id = gpg_id_file.read().strip()
-        else:
-            raise Exception("could not find .gpg-id file")
+        # Check if a main .gpg-id exists
+        self._get_gpg_id(self.path)
```

Comment:
We call this function only so that it raises if no gpg-id is found?

Reply:
Yeah, at initialization of the Password Store class to ensure there is a gpg-id at the root of the password store (to keep the existing behaviour/sanity-check there)